vocode/streaming/synthesizer/cartesia_synthesizer.py

```diff
@@ -0,0 +1,73 @@
+import io
+import wave
+
+from vocode import getenv
+from vocode.streaming.models.message import BaseMessage
+from vocode.streaming.models.synthesizer import CartesiaSynthesizerConfig
+from vocode.streaming.synthesizer.base_synthesizer import BaseSynthesizer, SynthesisResult
+
+
+class CartesiaSynthesizer(BaseSynthesizer[CartesiaSynthesizerConfig]):
+    def __init__(
+        self,
+        synthesizer_config: CartesiaSynthesizerConfig,
+    ):
+        super().__init__(synthesizer_config)
+        
+        # Lazy import the cartesia module
+        try:
+            from cartesia.tts import AsyncCartesiaTTS
+        except ImportError as e:
+            raise ImportError(
+                f"Missing required dependancies for CartesiaSynthesizer"
+            ) from e
+        
+        self.api_key = synthesizer_config.api_key or getenv("CARTESIA_API_KEY")
+        if not self.api_key:
+            raise ValueError("Missing Cartesia API key")
+        
+        self.cartesia_tts = AsyncCartesiaTTS
+        self.api_key = getenv("CARTESIA_API_KEY")
+        self.model_id = synthesizer_config.model_id
+        self.voice_id = synthesizer_config.voice_id
+        self.sampling_rate = synthesizer_config.sampling_rate
```

Comment:
@macwilk I just realized this is going to generate unintended consequences (specifically, slowed-down sounding audio) if we had initialized the Synthesizer with `from_telephone_output_device()` because it will be trying to pass `DEFAULT_SAMPLING_RATE` (8000).